packages/blocks/src/api/test/registration.js

```diff
@@ -1528,6 +1528,74 @@ describe( 'blocks', () => {
 			);
 		} );
 
+		// Check the `usesContext` array is correct.
+		it( 'should reject invalid usesContext property', () => {
+			registerBlockBindingsSource( {
+				name: 'core/testing',
+				label: 'testing',
+				usesContext: 'should be an array',
+			} );
+			expect( console ).toHaveWarnedWith(
+				'Block bindings source usesContext must be an array.'
+			);
+		} );
+
+		it( 'should add usesContext when only defined in the client', () => {
+			// Simulate server bootstrap.
+			registerBlockBindingsSource( {
+				name: 'core/testing',
+				label: 'testing',
+				usesContext: [ 'postId', 'postType' ],
+			} );
+			// Register source in the client without usesContext.
+			registerBlockBindingsSource( {
+				name: 'core/testing',
+				getValue: () => 'value',
+			} );
+			const source = getBlockBindingsSource( 'core/testing' );
+			unregisterBlockBindingsSource( 'core/testing' );
+			expect( source.usesContext ).toEqual( [ 'postId', 'postType' ] );
+		} );
+
+		it( 'should keep usesContext when it is not defined in the client', () => {
```

Comment:
And I believe this one should read, "should keep usesContext when only defined in the client".

Reply:
I will change the text.

Reply:
I made the change as part of [this commit](https://github.com/WordPress/gutenberg/pull/63513/commits/1d7160491a65c402a9b2ca5da6d4c1a87ee0b8c7).